podcasts/Sharing/Clip/VideoExporter.swift

```diff
@@ -0,0 +1,203 @@
+import SwiftUI
+import AVFoundation
+import UIKit
+import PocketCastsUtils
+
+protocol AnimatableContent: View {
+    func update(for progress: Double)
+}
+
+@available(iOS 16, *)
+class VideoExporter {
```

Comment:
Does it make sense for VideoExporter to be a class? from what I can see all it's methods are static. Maybe it can be an enum if you just using for to be a namespace?